In [9]:
import pandas as pd
import numpy as np
import seaborn as sb
import matplotlib.pyplot as plt
sb.set()

In [10]:
merged = pd.read_csv('merged.csv')
merged

Country  year  gdp_per_capita ($)  Total_Population  suicides_no  \
0        Albania  1987                 796           2709600           73   
1        Albania  1988                 769           2764300           63   
2        Albania  1989                 833           2803100           68   
3        Albania  1992                 251           2822500           47   
4        Albania  1993                 437           2807300           73   
...          ...   ...                 ...               ...          ...   
2316  Uzbekistan  2010                1533          25651783         1464   
2317  Uzbekistan  2011                1767          25978049         1640   
2318  Uzbekistan  2012                1964          26381830         1835   
2319  Uzbekistan  2013                2150          26838924         1950   
2320  Uzbekistan  2014                2309          27313507         2095   

      suicides/100k_pop  percentage of elderly(%)  Education index  \
0                  2.69                  2.118394              NaN   
1                  2.28                  2.123503              NaN   
2                  2.43                  2.122650              NaN   
3                  1.67                  2.217892            0.557   
4                  2.60                  2.261960            0.542   
...                 ...                       ...              ...   
2316               5.71                  2.023481            0.674   
2317               6.31                  2.061810            0.681   
2318               6.96                  2.093373            0.692   
2319               7.27                  2.109556            0.699   
2320               7.67                  2.099547            0.701   

      Inflation, consumer prices (annual %)  Unemployment Rate  
0                                       NaN                NaN  
1                                       NaN                NaN  
2                                       NaN                NaN  
3                                226.005421          30.014999  
4                                 85.004751          25.257999  
...                                     ...                ...  
2316                                    NaN           5.400000  
2317                                    NaN           5.000000  
2318                                    NaN           4.900000  
2319                                    NaN           4.900000  
2320                                    NaN           5.100000  

[2321 rows x 10 columns]

## Policy Analysis

Now that we have discovered that education index is the greatest contributor to the change in suicide rates, we decided to look zoom in on two different countries and how their policies affected their education index and overall suicide rates.

First we look at the correlation between education index and suicides per 100k population. 


In [11]:
correlation = merged.groupby('Country')[['suicides/100k_pop','Education index']].corr()

correlation = correlation.reset_index()

correlation= correlation.drop(columns=["level_1", "suicides/100k_pop"])

correlation



Country  Education index
0                Albania        -0.022644
1                Albania         1.000000
2    Antigua and Barbuda         0.004615
3    Antigua and Barbuda         1.000000
4              Argentina         0.594827
..                   ...              ...
197        United States         1.000000
198              Uruguay         0.766033
199              Uruguay         1.000000
200           Uzbekistan        -0.252281
201           Uzbekistan         1.000000

[202 rows x 2 columns]

In [12]:
correlation= correlation[correlation.index % 2 == 0]
correlation

correlation = correlation.reset_index()
correlation=correlation.drop(columns=["index"])
correlation=correlation.dropna()
correlation=correlation.sort_values(["Education index"], ascending=False)
correlation=correlation.rename(columns={"Education index": "Corr suicide rate/education index"})
correlation


Country  Corr suicide rate/education index
14  Bosnia and Herzegovina                           1.000000
58                  Mexico                           0.956857
73       Republic of Korea                           0.945125
15                  Brazil                           0.936263
24                  Cyprus                           0.928432
..                     ...                                ...
35                 Germany                          -0.962258
32                 Finland                          -0.965625
51                  Latvia                          -0.967904
30                 Estonia                          -0.980657
65                    Oman                          -0.999765

[90 rows x 2 columns]

## Policy 1

Bosnia and Herzegovina has the highest correlation (1) between suicide rates and education index, meaning a 1% increase in education index results in a 1% increase in suicide rate. However, the data only includes 2011 and 2014, making the results unreliable due to the small sample size.

In that regard, Mexico has the second highest correlation and has sufficient data from 1991 to 2015. Thus using Mexico as reference, we will be exploring how a policy in Mexico affects sucide rates.

Mexico created the Fund for Education and Payroll Operating Expenses, FONE for short, which was officially introduced in 2015. FONE aims to reallocate funds more efficiently and equitably to allow for greater funding to reach the most disadvantaged regions. It will include funding education services and teachers colleges.

After this policy was introduced in 2015, This surprisingly seems to decrease the sucide rate for teachers in the education system as most teachers not only felt that they were underpaid but that they were also unqualified to teach a huge class of students. After receiving a fair amount of pay, teachers would likely be more contempt with their job, and together with the increase in skill level of teachers, students will have a safer and more effective environment to study in. This reduces stress levels and the spillover effect would be the decrease in sucicide rates for students. 


https://www.oecd.org/education/Education-Policy-Outlook-Country-Profile-Mexico-2018.pdf



## Calculation of percentage increase/decrease of education index and suicide rates for Policy 1 (Mexico)


In [13]:
percent_increase_mexico_education=((merged["Education index"].iloc[1412]/merged["Education index"].iloc[1411])-1)*100
percent_increase_mexico_education


1.6641452344931862

Mexico's education index rose by 1.66% in 2015.


In [15]:
percent_increase_mexico_suicide_2013_2014=((merged["suicides/100k_pop"].iloc[1411]/merged["suicides/100k_pop"].iloc[1410])-1)*100
print(percent_increase_mexico_suicide_2013_2014)

percent_increase_mexico_suicide_2014_2015=((merged["suicides/100k_pop"].iloc[1412]/merged["suicides/100k_pop"].iloc[1411])-1)*100
percent_increase_mexico_suicide_2014_2015


6.61478599221792


-0.5474452554744547

This is accompanied by a 0.55% decrease in suicides in 2015, which is remarkably lower than the previous increase of 6.61%.

The suicide rate per 100k population went from postive in the previous to negative in 2015! This shows the effectiveness of the policy in bringing down suicides for countries with an education index like Mexico.


## Policy 2 


Next, Oman has the highest negative correlation (--0.999765) between suicide rates and education index, meaning a 1% increase in education index results in a 1% decrease in suicide rate. However, we once again do not have enough data, making the results unreliable.

Thus we look at the second highest negative correlation, Estonia.

To further insure the goal of language unification, the education law of 1993 made Estonian the sole language of instruction in all secondary schools by 2000. It seems that due to the full establishment of Estonian language being established as the sole language of instruction in all secondary school, children have been able to bring home what they learn in school to their parents, effectively teaching them Estonian as well. This is especially effective in teaching foreigners in Estonia, which are mainly Russians, Estonian, allowing the children of immigrants to coach their parents in how to live here, mediating their contacts with outsiders and forging ahead with language skills. With ways to interact better with the locals, there will be lesser discrimination and bullying, which will lead to smaller suicide rates.


https://journals.sagepub.com/doi/pdf/10.2304/csee.2004.6.2.101


## Calculation of percentage increase/decrease of education index and suicide rates for Policy 2 (Estonia)


In [16]:
percent_increase_estonia_education=((merged["Education index"].iloc[702]/merged["Education index"].iloc[701])-1)*100
percent_increase_estonia_education


3.1928480204342247

Estonia's education index rose by 3.19% in 2000.


In [17]:
percent_increase_estonia_suicide_1998_1999=((merged["suicides/100k_pop"].iloc[701]/merged["suicides/100k_pop"].iloc[700])-1)*100
print(percent_increase_estonia_suicide_1998_1999)

percent_increase_estonia_suicide_1999_2000=((merged["suicides/100k_pop"].iloc[702]/merged["suicides/100k_pop"].iloc[701])-1)*100
percent_increase_estonia_suicide_1999_2000


-1.2468827930174675


-20.707070707070706

This is accompanied by a 20.7% decrease in suicides in 2000, which is significantly lower than the previous decrease of 1.25%.

The decrease in suicides is a big sign that this policy is extremely effective in decreasing suicide rates for countries like Estonia.



## Conclusion

Overall, we have concluded that these 2 policies would be fitted to be used in countries with similar situations to Mexico and Estonia respectively to effectively reduce suicide rates. Countries with a similar education index for the case of Mexico, and for the case of Estonia, the presence of many foreigners and refugees in a country. 